app/views/applications/_form.html.erb

```diff
@@ -74,7 +74,8 @@
       {
         label: "Freeze deployments?",
         value: "1",
-        checked: @application.deploy_freeze?
+        checked: @application.deploy_freeze?,
+        hint: "Disables automatic deployments. Our deploy jobs will query the value of this flag in the API and abort if it is set. Manual deploy job builds will continue to work."
```

Comment:
Nit: Maybe it's clearer to just rename the option "Disable automatic deployments?"

Reply:
Thanks for the review. I decided against this, because the disable behaviour is just something this app _hopes_ will happen, when something else reads its API. The other reason is that it's more clearly applicable to someone who lands on the form in the context of a "deploy freeze", where they may not think "disable automatic deployments" is relevant.